tests/test_transform.py

```diff
@@ -39,3 +45,26 @@ def test_pairwiseSeq_2():
     test_list = [1, 2, 3, 4]
     result = distill.pairwiseSeq(test_list, split=True)
     assert result == ((1, 2, 3), (2, 3, 4))
+    
+def test_label_features_1():
+    def input_rule(log):
+        return "target" in log and "input" in log["target"]
+    logs = testing_utils.setup("./data/task_example.json", "datetime")
+    result = distill.label_features(logs,[FeatureDefinition(rule=input_rule, label="input_target")])
+    assert isinstance(result, list)
+    assert "input_target" in result 
```

Comment:
Doesn't label features return a list of logs? If so, this will fail. As it's going to check whether a string is in the list of logs, but only dictionaries are in it.

Instead, what you need to do is look at your input logs in `task_example.json`, figure out the indices of the logs that your FeatureDefintion will add the input_target label to, select those logs from the returned lists and validate that "input_target" has been added to the list stored in the "labels" field.